adapters/feeder2core/feeder2core.go

```diff
@@ -245,14 +246,58 @@ func AdaptCairo1Class(response *feeder.SierraDefinition, compiledClass json.RawM
 		class.EntryPoints.Constructor[index] = core.SierraEntryPoint{Index: v.Index, Selector: v.Selector}
 	}
 
-	if compiledClass != nil {
-		if err = json.Unmarshal(compiledClass, &class.Compiled); err != nil {
-			return nil, err
-		}
+	if class.Compiled, err = AdaptCompiledClass(compiledClass); err != nil {
+		return nil, err
 	}
+
 	return class, nil
 }
 
+func AdaptCompiledClass(compiledClass *feeder.CompiledClass) (core.CompiledClass, error) {
+	compiled := core.CompiledClass{}
+	compiled.Bytecode = compiledClass.Bytecode
+	pyHints, err := json.Marshal(compiledClass.PythonicHints)
```

Comment:
Can we let `compiled.PythonicHints` be `any` so we don't need to marshal here?

Reply:
As far as I know, `json.Marshal` doesn't accept types `any` or map[interface{}]interface{}. And we need to marshal `Compiled` field in `vm.Class`

Reply:
Marshal accepts `any` ([docs](https://pkg.go.dev/encoding/json#Marshal)). Did you mean something else?

Reply:
I had to clarify better: when marshling nested types like structs, json.Marshal can't marshal fields of type any as they are unsupported

Reply:
It [seems to work](https://play.golang.com/p/6aJgvJJkkeJ).